src/ptrace/_UPT_reg_offset.c

```diff
@@ -709,6 +709,39 @@ const int _UPT_reg_offset[UNW_REG_LAST + 1] =
     [UNW_S390X_F14]     = 0x150,
     [UNW_S390X_F15]     = 0x150,
     [UNW_S390X_IP]      = 0x08
+#elif defined(UNW_TARGET_RISCV64)
+    [UNW_RISCV64_PC]    = 0x0,
```

Comment:
Are there ever cases where this array is indexed with UNW_RISCV64_X0?

Reply:
Oops, missed this comment. Will add a check in _UPT_access_reg.